### PR TITLE
[ET-985] Hide view orders button from RSVP only events

### PR DESCRIPTION
**Pull request by rafsuntaskin**
[ET-985]

**RSVP only events before the change:**

Screen Shot 2021-03-09 at 4 41 54 PM

**RSVP only events after the change:**

Screen Shot 2021-03-09 at 4 42 32 PM

[ET-985]: https://theeventscalendar.atlassian.net/browse/ET-985

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -178,7 +178,7 @@ Check out our extensive [knowledgebase](https://evnt.is/18wm) for articles on us
 
 = [5.1.2] TBD =
 
-
+* Fix - Don't show View Orders link for RSVP only events. [ET-985]
```
Comment (sc0ttkclark) [inline readme.txt:181]: Describe where the view orders link is shown for context

**Review by sc0ttkclark — COMMENTED** [review]
Changelog needs more detail

File: src/admin-views/editor/button-view-orders.php [hunk 2]
```diff
@@ -1,11 +1,19 @@
 <?php
 $post_id = get_the_ID();
 
+// Get all the event providers.
+$providers = Tribe__Tickets__Tickets::get_active_providers_for_post( $post_id );
+
+// If RSVP is the only provider, bail out.
+if ( count( $providers ) == 1 && isset( $providers['Tribe__Tickets__RSVP'] ) ) {
```
Comment (juanfra) [inline src/admin-views/editor/button-view-orders.php:8]: Let's have a yoda conditional and strict comparison for the first one.

**Review by juanfra — CHANGES_REQUESTED** [review]

**Review by sc0ttkclark — DISMISSED** [review]

File: src/admin-views/editor/button-view-orders.php (hunk 2 above)
Comment (sc0ttkclark) [reply src/admin-views/editor/button-view-orders.php:8]:
```suggestion
if ( 1 === count( $providers ) && isset( $providers['Tribe__Tickets__RSVP'] ) ) {
```

**Review by sc0ttkclark — COMMENTED** [review]
Needs logic tweak

**Review by sc0ttkclark — APPROVED** [review]

**Pull request merged**